Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,36 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+           string[] firstName = { "Josie", "Daniel", "Liam" };
```

Comment:
The heights, weights and names are hardcoded. Could you make it more like it was required- by reading user input?